### PR TITLE
Real access implementation

### PR DESCRIPTION
**Pull request by georgecrawford**
## This is not yet ready for merge.

screenshot 2016-02-25 17 53 44
screenshot 2016-02-25 17 54 28
screenshot 2016-02-25 17 54 33

### CODE REVIEW

File: scss/style.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@ $o-grid-ie8-rules: 'none';
 @import 'o-grid/main';
 
 @include oFontsInclude(MetricWeb, light); // body
+@include oFontsInclude(MetricWeb, bold); // headings
```
Comment (georgecrawford) [inline scss/style.scss]: NB: this is a proposal without much thought, but it certainly improves the page design, and makes the CTA pop out. I'd also use it in other places.

Comment (quarterto) [inline scss/style.scss]: I've been meaning to do this for a while. It bumps us up from 150k (~30%) fonts to 200k (~40%). If we're doing this, I might consider nixing Financier Light Italic.

Comment (quarterto) [inline scss/style.scss]: Looking good btw, I especially like the note of teal.

File: scss/style.scss [hunk 2]
```diff
@@ -61,6 +62,20 @@ body {
 	border-bottom: $border-divider;
 }
 
+.headline.headline {
```
Comment (quarterto) [inline scss/style.scss]: I assume the double classname is to override `oTypographyBodyWrapper`?

Comment (georgecrawford) [inline scss/style.scss]: Exactly - I'll add a comment

**Comment by georgecrawford** [comment]
Replacing this with https://github.com/Financial-Times/google-amp/pull/44

**Pull request closed without merging**